migration/v0.7.x-v0.8.x/migrate-prometheus-operator.md

```diff
@@ -0,0 +1,37 @@
+The `prometheus-operator` chart has been replaced with `kube-prometheus-stack`. When upgrading, you must remove your deployed `prometheus-operator` resources before applying the new chart.
+
+```
+# Workload cluster
+
+# Delete old release and related resources
+./bin/ck8s ops helm wc uninstall prometheus-operator -n monitoring
+./bin/ck8s ops kubectl wc delete service/prometheus-operator-kubelet -n kube-system
+./bin/ck8s ops kubectl wc delete pvc/prometheus-prometheus-operator-prometheus-db-prometheus-prometheus-operator-prometheus-0 -n monitoring
+
+# Bootstrap new CRDs
+./bin/ck8s bootstrap wc
+
+#
+# Update falco.alerts.hostPort to "http://kube-prometheus-stack-alertmanager.monitoring:9093" in ${CK8S_CONFIG_PATH}/wc-config.yaml
+#
+# Update falco_alerts_alertmanager_hostport to "http://kube-prometheus-stack-alertmanager.monitoring:9093" in ${CK8S_CONFIG_PATH}/config.yaml
```

Comment:
This line looks like it's from an old config.
```suggestion
```

Reply:
Alright, fixed that.